python/contrib/dag/__init__.py

```diff
@@ -33,22 +33,6 @@ def __cacheGraph(self):
             self.__cached_graph = self.clone()
         return self.__cached_graph
 
-    # Added by the MOOSE group
```

Comment:
I'm going to suggest that we don't remove this method. I'm **still** not convinced that we don't need it if we are doing things right. When I have some time, I'd like to sit down and really go through a few scenarios of the DAG. I've thought about this a lot and don't see why we don't need this method. Just leave it in for now in case we decide we'll want it later.